zppy/defaults/default.ini

```diff
@@ -336,17 +344,34 @@ nrows = integer(default=4)
 # net_atm_water_imbalance requires PRECC, PRECL, QFLX
 # Remove the 3 ocean plots (change_ohc,max_moc,change_sea_level) if you don't have ocean data.
 plots_original = string(default="net_toa_flux_restom,global_surface_air_temperature,toa_radiation,net_atm_energy_imbalance,change_ohc,max_moc,change_sea_level,net_atm_water_imbalance")
+#
+# These are only needed if make_viewer = True:
+#
 # The names of the extra plots you want displayed (i.e., define specific variables)
 # These should be a subset of the `vars` generated by the `ts` `glb` subtasks.
 plots_atm = string(default="")
 plots_ice = string(default="")
 # Set `plots_lnd = "all"` to run every variable in the land csv file.
 plots_lnd = string(default="")
 plots_ocn = string(default="")
-# regions to plot: glb, n, s (global, northern hemisphere, southern hemisphere)
-regions = string(default="glb,n,s")
-ts_years = string_list(default=list(""))
-# `years = "1-100",` would plot years 1 to 100 on the graphs.
+
+  [[__many__]]
```

Comment:
Adds `global_time_series` subtask functionality. Per the legacy testing, not having a subsection does _not_ cause a failure.